test/WebSites/JsonPatchWebSite/ProductCategoryConverter.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using JsonPatchWebSite.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace JsonPatchWebSite
+{
+    public class ProductCategoryConverter : JsonConverter
+    {
+        public override bool CanConvert(Type objectType)
+        {
+            return true;
+        }
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            var obj = JObject.Load(reader);
```

Comment:
Yes but since reader has value and if I don't load it, then I was getting an error "Additional text found in JSON string after finishing deserializing object."